generator/testdata/rust/protobuf/golden/location/src/model.rs

```diff
@@ -134,7 +134,8 @@ impl gax::paginator::PageableResponse for ListLocationsResponse {
     }
 
     fn next_page_token(&self) -> std::string::String {
-        gax::paginator::extract_token(&self.next_page_token)
+        use std::clone::Clone;
+        self.next_page_token.clone()
```

Comment:
I think you could write:

```suggestion
        std::clone::Clone::clone(&self.next_page_token)
```

But I am not sure that is better in any way.